pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Discover the new features, updates, and known limitations in this release of the Elastic Service
+title: Release Notes
+menuWeight: 10
+model: /mesosphere/dcos/services/elastic/data.yml
+render: mustache
+---
+
+# Release notes for Elastic Service version 3.1.0-7.4.1
+
+- [Upgrades](#orgdf76f77)
+
+<a id="orgdf76f77"></a>
+
+## Upgrades
+
+-   Upgraded Elasticsearch and Kibana to [7.4.1](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/release-notes-7.4.1.html)
```

Comment:
Since this is the Elastic 7 line, I think it would be good to include the Release Notes from the Elastic 3.0.0-7.3.2 package too below this section